cluster-autoscaler/core/scaledown/actuation/drain.go

```diff
@@ -27,6 +27,7 @@ import (
 	policyv1beta1 "k8s.io/api/policy/v1beta1"
 	kube_errors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/autoscaler/cluster-autoscaler/core/scaledown/deletiontracker"
```

Comment:
I would consider dropping dependency on deletion tracker and just accepting a simple local interface in `NewDefaultEvictor`.

Reply:
to double check I understood it correctly: you would like me to introduce a new local interface with RegisterEviction function (implemented by node deletion tracker)?

Reply:
created new interface